api/src/Filter/ArchivedFilter.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filter;
+
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\AbstractFilter;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use App\Entity\ArchivableInterface;
+use Doctrine\ORM\QueryBuilder;
+
+final class ArchivedFilter extends AbstractFilter implements GenericFilterInterface
```

Comment:
No need to extend AbstractFilter if you redefine `apply` function, prefer using the following syntax and create a totally new filter:
```php
final class ArchivedFilter implements FilterInterface
```

Reply:
But `FilterInterface` doesn't expose the `apply` function. When only implementing `FilterInterface` the filter isn't autoloaded anymore.

Reply:
The FilterInterface do expose the `apply` function, check the right one: https://github.com/api-platform/core/blob/master/src/Bridge/Doctrine/Orm/Filter/FilterInterface.php

Reply:
Ha yes, I was using the wrong one.